src/main/uwsgi.ini

```diff
@@ -0,0 +1,14 @@
+[uwsgi]
```

Comment:
Ik zag net je PR in openstack, daar haal je deze UWSGI config weg:

```yaml
        UWSGI_BUFFER_SIZE: "32768"
        UWSGI_HARAKIRI: "15"
        UWSGI_HTTP: ":8000"
        UWSGI_MASTER: "1"
        UWSGI_MODULE: "stadsarchief.wsgi:application"
        UWSGI_PROCESSES: "4"
        UWSGI_STATIC_MAP: "/static=/static"
        UWSGI_VACUUM: "1
```

Moet je die hier nog terugplaatsen?

Reply:
Harakiri en buffer_size are back. Thanks